lit_gpt/prompts.py

```diff
@@ -19,6 +23,10 @@ def stop_tokens(self, tokenizer: "Tokenizer") -> Tuple[List[int], ...]:
     def from_name(cls, name: str) -> "PromptStyle":
         return prompt_styles[name]()
 
+    @classmethod
+    def from_config(cls, config: Config) -> "PromptStyle":
+        return model_name_to_prompt_style(config.name)
+
 
 class Default(PromptStyle):
```

Comment:
For `Alpaca`, why did you use `kwargs.get("input")` instead of adding input the the `apply` signature?

Reply:
I'm not sure, I thought I had to keep the `**kwargs` as a requirement from the typsystem. I'll check if it works

Reply:
I know why I did it this way. It's so that I can conveniently pass the "remainder" of the sample data with arbitrary keys:
https://github.com/Lightning-AI/lit-gpt/blob/07d34733623cbcfbff345463c3735f83f91f6551/lit_gpt/data/base.py#L83

This way, custom datasets can have additional rows and a prompt style that defines a variable number of them. So I don't need to hardcode "inputs" in the base class or condition it on a specific prompt style class.